From 870529ca8a7411842caee6e438f10f5665a55332 Mon Sep 17 00:00:00 2001
From: Joel Galenson <jgalenson@google.com>
Date: Mon, 26 Mar 2018 16:37:42 -0700
Subject: [PATCH 8/8] Hide some denials.

These denials occur fairly often, causing some logspam.

Bug: 77225170
Test: Boot device.
Merged-In: Icd73a992aee44007d0873743f706758f9a19a112
Change-Id: Icd73a992aee44007d0873743f706758f9a19a112
(cherry picked from commit a66d1a4543fc6d43fe2e798bb81347d1097227cf)
---
 private/untrusted_app_all.te | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/private/untrusted_app_all.te b/private/untrusted_app_all.te
index cce589ea..a090ec13 100644
--- a/private/untrusted_app_all.te
+++ b/private/untrusted_app_all.te
@@ -106,3 +106,10 @@ allow untrusted_app_all preloads_data_file:dir search;
 allow untrusted_app_all vendor_app_file:dir { open getattr read search };
 allow untrusted_app_all vendor_app_file:file { open getattr read execute };
 allow untrusted_app_all vendor_app_file:lnk_file { open getattr read };
+
+# This is allowed for targetSdkVersion <= 25 but disallowed on newer versions.
+dontaudit untrusted_app_all net_dns_prop:file read;
+
+# This has been disallowed since Android O.
+# For P, we assume that apps are safely handling the denial.
+dontaudit untrusted_app_all proc_stat:file read;
-- 
2.17.0

